scst-store/deployment_details.md

```diff
@@ -58,3 +58,7 @@ app_service_type: "LoadBalancer"
 By default, a service account with read-write privileges to the metadata store app is installed. This service account is a cluster-wide account that uses ClusterRole. If the service account and role are not desired, set the `add_default_rw_service_account` property to `"false"`. To create a custom service account, see [create service account](create_service_account_access_token.md).
 
 The store will automatically create a read-only cluster role, which may be bound to a service account via `ClusterRoleBinding`. To create service accounts to bind to this cluster role, see [create service account](create_service_account_access_token.md). 
+
+## Exporting Certificates
+
+The Store creates [Secret Export](https://github.com/vmware-tanzu/carvel-secretgen-controller/blob/develop/docs/secret-export.md) for exporting certain certificates to the `Supply Chain Security Tools for VMware Tanzu - Scan` so that it can securely post scan results to the Store. These certs are exported to the namespace the `Scan` is installed to. 
```

Comment:
Don't use the term _The Store_. I know it's been used in the past, but I'm trying to correct it. We got feedback from the docs team to use only marketing approved terms. So I only use either "Supply Chain Security Tools - Store" or very generic terms such as "It" or "API server" or "The deployment".